docs/en/engines/table-engines/special/keepermap.md

```diff
@@ -105,6 +107,7 @@ TRUNCATE TABLE keeper_map_table;
 ### Updates
 
 Values can be updated using `ALTER TABLE` query. Primary key cannot be updated.
+If setting `keeper_map_strict_mode` is set to `true`, fetching and updating data will succeed only if it's executed atomically.
```

Comment:
To be sure: we execute ALTER in one `mutate` call and update actual values in `sink->finalize<true>` call. That's why updating one chunk of data and getting an exception later is impossible, is it?

Reply:
yes, we actualy send requests only on `finalize`.
Until then we fetch and modify data locally.